### PR TITLE
flightgear & simgear: 2018.2.2 -> 2018.3.1

### PR DESCRIPTION
**Pull request by aanderse**
###### Motivation for this change
https://github.com/NixOS/nixpkgs/pull/52020

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/games/flightgear/default.nix [hunk 1]
```diff
@@ -6,8 +6,8 @@
 }:
 
 let
-  version = "2018.2.2";
-  shortVersion = "2018.2";
+  version = "2018.3.1";
+  shortVersion = "2018.3";
```
Comment (worldofpeace) [inline pkgs/games/flightgear/default.nix]: Shouldn't there be a hash change below?

Comment (aanderse) [reply pkgs/games/flightgear/default.nix]: Should have marked WIP, going through build steps, still need to push :smile:

**Comment by aanderse** [comment]
@teto this should build for you now, at least on x86_64-linux...

**Comment by teto** [comment]
Why the ping ? because I subscribed to notifications xD ? I can try to build it if you want.

**Comment by aanderse** [comment]
> Why the ping ? because I subscribed to notifications xD ? I can try to build it if you want.

oops! totally meant for @timokau and not you. sorry for confusion!

**Comment by 7c6f434c** [comment]
@GrahamcOfBorg build flightgear

**Review by matthiasbeyer — APPROVED** [review]

**Comment by aanderse** [comment]
> @GrahamcOfBorg build flightgear

After seeing how long it took on my machine I didn't even bother ...

**Comment by 7c6f434c** [comment]
Well, the timeout was way later than the failures in the previous updates, so I did receive a confirmation that the previous ofBorg issues were not unique (an unsolved even by double update). Thanks.

**Pull request merged**